src/content/guides/production.md

```diff
@@ -175,6 +176,7 @@ __webpack.prod.js__
   })
 ```
 
+Note: _We have not used `inline-source-map` because it's not optimized for production and would generate a giant bundle, and so reduce the overall performances. Same goes for `devtool`s that contain `eval` in the name._
```

Comment:
Hmmm I would use our custom tip syntax and tweak the wording a bit from what was used in the tweet, e.g.:

``` md
T> Avoid `inline-***` and `eval-***` use in production as they can increase bundle size and reduce the overall performance.
```

Reply:
Got it :) I'll update it now